### PR TITLE
Adding a getOutcome method to Message\Response to solve Issue 106

### PR DESCRIPTION
**Pull request by phpguru**
Solves https://github.com/thephpleague/omnipay-stripe/issues/106

### CODE REVIEW

File: src/Message/Response.php [hunk 1]
```diff
@@ -55,13 +55,29 @@ public function isSuccessful()
      */
     public function getChargeReference()
     {
-        if (isset($this->data['object']) && $this->data['object'] == 'charge') {
+        if (isset($this->data['object']) && 'charge' === $this->data['object']) {
```
Comment (phpguru) [inline src/Message/Response.php]: consistency with other methods

**Comment by barryvdh** [comment]
Can you fix the codestyle, so we can merge this? Thanks!

**Comment by phpguru** [comment]
Sorry, I'm not seeing any differences in code style. Happy to correct it - can you let me know what you need?

**Comment by barryvdh** [comment]
See https://travis-ci.org/thephpleague/omnipay-stripe/jobs/390425945

```
FILE: ...ravis/build/thephpleague/omnipay-stripe/src/Message/Response.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 67 | ERROR | [x] Whitespace found at end of line
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by barryvdh** [comment]
Much better :)

**Comment by phpguru** [comment]
Gotcha! Thanks for the link. Should be good to go now.

**Pull request merged**